Fitness/Controllers/HabitTrackingController.swift

```diff
@@ -549,7 +573,14 @@ extension HabitTrackingController: HabitTrackerOnboardingDelegate {
     
     func swipeRight() {
         swipedCell = nil
-        habitTableView.separatorStyle = .singleLine
+        
+        for indexPath in habitTableView.indexPathsForVisibleRows ?? [] {
+            if let cell = habitTableView.cellForRow(at: indexPath) as? HabitTrackerOnboardingCell {
+                if indexPath.section == 1 {
```

Comment:
Can we pull this constant 1 up to the FileConstants enum as well? I think it'll make the design more clear if it ever needs to be changed.

Reply:
sounds good